charts/airbyte/templates/airbyte-db.yaml

```diff
@@ -43,9 +43,15 @@ spec:
             - name: POSTGRES_DB
               value: {{ .Values.postgresql.postgresqlDatabase }}
             - name: POSTGRES_PASSWORD
-              value: {{ .Values.postgresql.postgresqlPassword }}
+              valueFrom:
+                secretKeyRef:
+                  name: airbyte-airbyte-secrets
```

Comment:
What is the naming scheme here?

Reply:
It's `<release>-airbyte-secrets` where `<release>` is also `airbyte`. It was too much trouble changing it everywhere.

Reply:
can that be Values.golobal.database.secretName ?